src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -927,6 +927,12 @@
   <data name="ERR_FixedNotNeeded" xml:space="preserve">
     <value>You cannot use the fixed statement to take the address of an already fixed expression</value>
   </data>
+  <data name="ERR_ExprCannotBeFixed" xml:space="preserve">
+    <value>The given expression cannot be used in a fixed statement</value>
+  </data>
+  <data name="ERR_FixableHelperDoesNotFitThePattern" xml:space="preserve">
+    <value>The method '{0}' does not meet requirements necessary </value>
```

Comment:
>The method '{0}' does not meet requirements necessary [](start = 11, length = 53)

I feel this message should give more information about what is expected.

Reply:
Logged an issue to follow up with more detailed diagnostics - [https://github.com/dotnet/roslyn/issues/24947](https://github.com/dotnet/roslyn/issues/24947)


---
In reply to: [167650429](https://github.com/dotnet/roslyn/pull/24469#discussion_r167650429) [](ancestors = 167650429)